R/CohortRelationship.R

```diff
@@ -327,25 +201,60 @@ executeCohortRelationshipDiagnostics <- function(connection,
                                                  temporalCovariateSettings,
                                                  minCellCount,
                                                  recordKeepingFile,
-                                                 incremental) {
+                                                 incremental,
+                                                 batchSize = 500) {
   ParallelLogger::logInfo("Computing Cohort Relationship")
   startCohortRelationship <- Sys.time()
-
-  subset <- subsetToRequiredCohorts(
-    cohorts = cohortDefinitionSet,
+  
+  allCohortIds <- cohortDefinitionSet %>%
+    dplyr::select(.data$cohortId, .data$checksum) %>%
+    dplyr::rename(targetCohortId = .data$cohortId,
+                  targetChecksum = .data$checksum) %>%
+    dplyr::distinct()
+  combinationsOfPossibleCohortRelationships <- allCohortIds %>%
```

Comment:
Reduce the length of this variable name. Use comments for this kind of explanation as it's clearer

Reply:
Why reduce the length of the variable? Is there a limit?